### ISSUE TITLE
docker-compose up not attaching to container

### ISSUE THREAD

**Issue by nhumrich** [docker/compose#1961]
I cant seam to run anything with docker-compose now that I am using the development version.

    $ docker-compose --version
    docker-compose version: 1.5.0dev
  
    $ git log --oneline --decorate
    83c514f (origin/master, origin/HEAD) Merge pull request #1933 from dnephin/fix_scale_test_flakes
    235fe21 Prevent flaky test by changing container names.

Trying to run `up` spins up the container, then immediately closes it:

    $ docker-compose up
    Recreating smeservice_web_1...
    Attaching to 
    Gracefully stopping... (press Ctrl+C again to force)
    Stopping smeservice_web_1...

running 
    docker run -it busybox /bin/sh -c "while true; do /bin/echo 'hello'; sleep 5; done;"

works fine.

Here is my docker-compose.yml

    web:
      image: busybox
      command: /bin/sh -c "while true; do /bin/echo 'hello'; sleep 5; done;"

Running on version 1.4.0 gives me the following:

    $ docker-compose up
    Starting smeservice_web_1...
    Attaching to smeservice_web_1 
    web_1 | hello
    web_1 | hello

Other info:

    $ python --version
    Python 3.4.0

Running Ubuntu 14.10

    $ uname -r
    3.19.0-26-generic

**Comment by dnephin** [comment]
I think I might have broken this with the changes in #1931 and we're missing some test coverage for it.  I'll look into it today.

**Comment by Silex** [comment]
Seems to be broken in 1.4.1 as well:

```
philippe@philippe-desktop:~/work/stvs/provision$ docker-compose up
Creating provision_iso_1...
Creating provision_old_1...
Creating provision_test_1...
Attaching to 
^CGracefully stopping... (press Ctrl+C again to force)
Stopping provision_test_1... 
Stopping provision_test_1... done
Stopping provision_iso_1... done

Aborting.
```

Waited 2 minutes before pressing `C-c` after "Attaching to" said nothing more. The containers were spinning and `docker-compose logs` worked from another terminal tho.

**Comment by dnephin** [comment]
The fix for this is in 1.5.0

**Comment by brunowego** [comment]
+1
